docs/docsite/rst/roadmap/COLLECTIONS_7.rst

```diff
@@ -0,0 +1,56 @@
+.. _ansible_7_roadmap:
+
+===================
+Ansible project 7.0
+===================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-core <https://pypi.org/project/ansible-core/>`_ package as well. All dates are subject to change. See the `ansible-core 2.14 Roadmap <https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_14.html>`_ for the most recent updates on ``ansible-core``.
+
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2022-09-19: ansible-core feature freeze, stable-2.14 branch created.
+:2022-09-26: Start of ansible-core 2.14 betas (biweekly, as needed).
+:2022-09-27: Ansible-7.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.14`` pre-releases).
+:2022-10-12: Community Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
+:2022-10-17: First ansible-core 2.14 release candidate.
+:2022-10-18: Ansible-7.0.0 alpha2.
+:2022-10-26: Community Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
+:2022-11-07: Ansible-core-2.14 released.
+:2022-11-08: Ansible-7.0.0 alpha3.
+:2022-11-14: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-7. This includes adding new collections to Ansible 7.0.0; from now on new collections have to wait for 7.1.0 or later.
+:2022-11-15: Create the ansible-build-data directory and files for Ansible-8.
+:2022-11-15: Ansible-7.0.0 beta1 -- feature freeze [1]_ (weekly beta releases; collection owners and interested users should test for bugs).
+:2022-11-22: Ansible-7.0.0 beta2.
+:2022-11-29: Ansible-7.0.0 rc1 [2]_ [3]_ (weekly release candidates as needed; test and alert us to any blocker bugs).  Blocker bugs will slip release.
+:2022-12-13: Ansible-7.0.0 release.
+:2022-01-03: Release of Ansible-7.1.0 (bugfix + compatible features: every three weeks.)
```

Comment:
That 6.0.0a3 was released after ansible-core 2.13 GA was an artefact of ansible-core 2.13 GA being released one week earlier. So if we stick to the "standard" cycle, we have to adjust for that:
```suggestion
:2022-11-07: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-7. This includes adding new collections to Ansible 7.0.0; from now on new collections have to wait for 7.1.0 or later.
:2022-11-08: Create the ansible-build-data directory and files for Ansible-8.
:2022-11-08: Ansible-7.0.0 beta1 -- feature freeze [1]_ (weekly beta releases; collection owners and interested users should test for bugs).
:2022-11-15: Ansible-7.0.0 beta2.
:2022-11-22: Ansible-7.0.0 rc1 [2]_ [3]_ (weekly release candidates as needed; test and alert us to any blocker bugs).  Blocker bugs will slip release.
:2022-12-06: Ansible-7.0.0 release.
:2022-12-27: Release of Ansible-7.1.0 (bugfix + compatible features: every three weeks.)
```

Reply:
The following proposal was accepted by the vote in https://github.com/ansible-community/community-topics/discussions/118:
```suggestion
:2022-11-07: Ansible-core-2.14.0 released.
:2022-11-07: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-7. This includes adding new collections to Ansible 7.0.0; from now on new collections have to wait for 7.1.0 or later.
:2022-11-08: Create the ansible-build-data directory and files for Ansible-8.
:2022-11-08: Ansible-7.0.0 beta1 -- feature freeze [1]_ (weekly beta releases; collection owners and interested users should test for bugs).
:2022-11-15: Ansible-7.0.0 rc1 [2]_ [3]_ (weekly release candidates as needed; test and alert us to any blocker bugs).  Blocker bugs will slip release.
:2022-11-29: Ansible-7.0.0 release.
:2022-12-06: Release of ansible-core 2.14.1.
:2022-12-20: Release of Ansible-7.1.0 (bugfix + compatible features: every three weeks.)
```